pkg/operator/ceph/config/monstore.go

```diff
@@ -192,9 +192,37 @@ func (m *MonStore) SetKeyValue(key, value string) error {
 }
 
 func (m *MonStore) SetAll(clientName string, settings map[string]string) error {
+	keys, err := m.setAll(clientName, settings)
+	if err != nil {
+		return errors.Wrapf(err, "failed to set all keys")
+	}
+	if len(keys) == 0 {
+		return nil
+	}
+	logger.Infof("failed to set keys %v, trying to remove them first\n", keys)
```

Comment:
shouldn't need a newline, the logger adds one automatically
```suggestion
	logger.Infof("failed to set keys %v, trying to remove them first", keys)
```